doc/Type/Hash.pod6

```diff
@@ -337,13 +337,17 @@ Example:
     push %h, 'd' => 5;              # a => [1,1,1,1,1], b => 3, c => 4, d => 5
 
 Please note that literal pairs in the argument list may be interpreted as
-L<named arguments|/type/Capture> and as such won't end up in the C<Hash>.
-The C<push> subroutine will throw if it receives a named argument.
+L<named arguments|/type/Capture> and as such won't end up in the C<Hash>:
 
     my %h .= push(e => 6);
-    try push %h, f => 7;
     say %h.raku; # OUTPUT: «{}␤»
 
+Use the corresponding L<subroutine|/language/independent-routines#sub_push> to
+catch this kind of mistake:
+
+    CATCH { default { put .message } };
+    push my %h, f => 7; # OUTPUT: «Unexpected named argument 'f' passed␤»
+
```

Comment:
I'm not totally sure this belongs here now. It should probably be moved to the new reference you have created.

Reply:
I thought it would be more useful here. I did it the other way around for 'append'. As both the method and the sub pages contain cross-references between `append` and `push` I thought this would be enough to let users figure out when to use the method (e.g. because they know what they're doing and want maximum efficiency) and when to use the subroutine.